settings.py

```diff
@@ -38,12 +36,13 @@
 DEV_MODE = os.environ['SERVER_SOFTWARE'].startswith('Development')
 UNIT_TEST_MODE = os.environ['SERVER_SOFTWARE'].startswith('test')
 
+#setting GOOGLE_CLOUD_PROJECT manually in dev mode
+APP_ID = os.environ.get('GOOGLE_CLOUD_PROJECT', 'dev') if DEV_MODE else os.environ['GOOGLE_CLOUD_PROJECT']
```

Comment:
@CrypticWizard, Tests on Travis CI are failing due to the following issue - if `DEV_MODE` is `False` and `UNIT_TEST_MODE` is` True`, `APP_ID` will be assigned using `os.environ['GOOGLE_CLOUD_PROJECT']` which will raise a `key error` since `GOOGLE_CLOUD_PROJECT` does not exist while running tests in Travis CI. Hence, I would recommend that you add `UNIT_TEST_MODE` in the condition using `OR` as shown below

`APP_ID = os.environ.get('GOOGLE_CLOUD_PROJECT', 'dev') if DEV_MODE or UNIT_TEST_MODE else os.environ['GOOGLE_CLOUD_PROJECT']`

Reply:
Done!